### ISSUE TITLE
[Epic] Better Integrate Session Replay in the Sentry Experience

### ISSUE THREAD

**Issue by jas-kas** [getsentry/sentry#50057]
Today, Session Replay is integrated into core workflows such as Issues (i.e., view replays associated with issues) and Performance (i.e., view replays associated with slow transactions).

This initiative is about better integrating Session Replay in other impactful touch points such as...

- Issue Alert Emails and Slack Notifications
- Weekly Reports
- Slack Issue Unfurls
- Project Creation Flow for AM1 Organizations

(Internal) More details can be found in this PRD: https://www.notion.so/sentry/FY-24-Q2-Better-Integrating-Replay-in-the-Sentry-Experience-cd32832e809b49c7a63ac6570c55dd69 (WIP)

**Comment by bruno-garcia** [comment]
See: https://github.com/getsentry/sentry/pull/51388

**Comment by therealarkin** [comment]
what's the latest here?

**Comment by lucaciraolo** [comment]
Hi, our team use replays mainly just for watching what our users are doing, i.e. the replays are not linked to an issue.
The difficulty we are facing is keeping track of which replays have been watched. It would be super useful if we could set up a slack integration which could notify a channel when there is a new replay. This way we could keep track of who has watched it + add comments etc.

TLDR: Feature request for slack notifications for sentry replays

**Comment by jas-kas** [comment]
Hi @lucaciraolo, thanks for your feedback and for sharing your Replay use-case!

Alert notifications for new replays are currently not planned; however, I made note of your feature request.

> It would be super useful if we could set up a slack integration which could notify a channel when there is a new replay. 
How many replays do you typically get in a week? Since there is a chance these notifs could get spammy, are there any alert configurations you'd like to have for 'new replay' alerts?

**Comment by ryan953** [comment]
Closing. This is long done, we including information about replay in the different emails, and links to replays from within the alerts.
